py/kubeflow/testing/new_test_util.py

```diff
@@ -0,0 +1,158 @@
+"""
+This module contains misc utils used for Kubeflow testing
+infrastructure. They include creating test cases/suites,
+managing logs of test cases and initializing tests
+
+This is intended to replace test_util.py
+"""
+import argparse
```

Comment:
Could we just call this package util.py?
The package is kubeflow.testing
so unless we already have a util.py file that seems fine. And this avoids the prefix "new".

We should avoid the prefix "new" because at some point we'd probably want to rename it anyway since the "new" will no longer be relevant and then we'd just have to update a bunch of call sites.

So it would be better to pick a name that we won't want to change.

Reply:
We already have a [util.py](https://github.com/kubeflow/testing/blob/master/py/kubeflow/testing/util.py)

Reply:
Renamed to `test_helper.py`